### ISSUE TITLE
Can't find Python globals when using the `tohil::call` command from tcl

### ISSUE THREAD

**Issue by NasaGeek** [flightaware/tohil#39]
```python
Python 3.7.6 (default, Jan 30 2020, 01:17:40)
[Clang 8.0.0 (tags/RELEASE_800/final 356365)] on freebsd11
Type "help", "copyright", "credits" or "license" for more information.
I >>> import tohil
I >>> tohil.package_require("tohil")
'4.0.0'
I >>> tohil.call("::tohil::call", "print", "hello world")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
tohil.TclError: ("module '__main__' has no attribute 'print'", <tohil.tcldict: '-code 1 -level 0 -errorcode {PYTHON AttributeError {module \'__main__\' has no attribute \'print\'}} -errorinfo {module \'__main__\' has no attribute \'print\' (failed to find object/function in python interpreter)\nfrom python code executed by tohil\n    invoked from within\n"::tohil::call print {hello world}"} -errorline 1'>)
```

This does work, though
```python
I >>> tohil.call("::tohil::eval", "print('hello world')")
hello world
<tohil.tclobj: ''>
```

Oh I just realized the bouncing back and forth is a red herring, this is failing just from `tclsh` as well
```tcl
 ~  tclsh
hopnu chris.roberts % package require tohil
4.0.0
hopnu chris.roberts % tohil::call print "hello world"
module '__main__' has no attribute 'print'
while evaluating tohil::call print "hello world"
```

**Comment by lehenbauer** [comment]
This works, not sure what it means just yet.  `tohil::call __builtins__.print "hello world"`

**Comment by lehenbauer** [comment]
This also "fixes the glitch" `tohil::exec "from builtins import *"`

**Comment by lehenbauer** [comment]
I suspect the code in TohilCall_Cmd that finds the callable to pass to PyObject_Call is clunky.  Yep, errorInfo includes `module '__main__' has no attribute 'print' (failed to find object/function in python interpreter)` which pins it down to PyObject_GetAttrString in the vicinity of line 759.

**Comment by lehenbauer** [comment]
Fixed by PR #42
